org.eclipse.jdt.core/model/org/eclipse/jdt/internal/core/util/messages.properties

```diff
@@ -19,6 +19,7 @@
 ### JavaModel messages.
 
 ### hierarchy
+AbstractImageBuilder_mr_missmatch_main=Target release for {0} ({1}) is lower or equal to default release ({2}).
```

Comment:
What is the meaning of "default" here?

Reply:
The default release is the one of the project (e.g. the usual code uses Java 8 (=default) where there is some code targeting Java 11 in a multi-release (=non default) way).

Reply:
Yes the JEP has the "advantage" that it only has to think about a final JAR where they can always assume everything as before and in contrast one has "versioned (class) entries", but this seems not suitable wording here. I came up with "default" because we have some folders that using the default project settings and others diverge in the aspect of the multi-release target.

Also in maven one has usually "default executions" of a mojo and then one configures additionally ones. In any case I would be open to suggestions how to better word this, the project should be quite clear because its an error marker on a project file.